Tests/Marketplace/marketplace_services.py

```diff
@@ -737,17 +737,25 @@ def sign_pack(self, signature_string=None):
         finally:
             return task_status
 
-    def encrypt_pack(self, zip_pack_path, pack_name, encryption_key, extract_destination_path):
+    @staticmethod
+    def encrypt_pack(zip_pack_path, pack_name, encryption_key, extract_destination_path):
         try:
+            current_working_dir = os.getcwd()
             shutil.copy('./encryptor', os.path.join(extract_destination_path, 'encryptor'))
             os.chmod(os.path.join(extract_destination_path, 'encryptor'), stat.S_IXOTH)
-            current_working_dir = os.getcwd()
             os.chdir(extract_destination_path)
             output_file = zip_pack_path.replace("_not_encrypted.zip", ".zip")
             subprocess.call('chmod +x ./encryptor', shell=True)
             full_command = f'./encryptor ./{pack_name}_not_encrypted.zip {output_file} "' \
                            f'{encryption_key}"'
+
             subprocess.call(full_command, shell=True)
+            new_artefacts = os.path.join(current_working_dir, 'private_artifacts')
+            if os.path.exists(new_artefacts):
+                shutil.rmtree(new_artefacts)
+            os.mkdir(path=new_artefacts)
+            shutil.copy(zip_pack_path, os.path.join(new_artefacts, f'{pack_name}_not_encrypted.zip'))
+            shutil.copy(output_file, os.path.join(new_artefacts, f'{pack_name}.zip'))
             os.chdir(current_working_dir)
         except subprocess.CalledProcessError as error:
             print(f"Error while trying to encrypt pack. {error}")
```

Comment:
```suggestion
        except subprocess.CalledProcessError as error:
            print(f"Error while trying to encrypt pack. {error}")
        except shutil.Error as error:
            do_x
```
I think it is worth handling shutil errors too